setup.cfg

```diff
@@ -19,6 +19,7 @@ install_requires =
     astropy >=3.0
     scipy
     Pillow
+    sphinx-astropy
```

Comment:
why was this moved from the docs dependencies into the main dependencies? seems like it's not a dependency that should be need to run astrocut

Reply:
`sphinx-astropy` was previously defined by the `docs` dependencies here: https://github.com/spacetelescope/astrocut/blob/main/setup.cfg#L33 

I think the issue might be that `alldeps` in tox is not pulling in the docs dependencies.

these _should_ be installed by tox under specific test configurations when testing the docs. It seems like they are not. With my limited experience with `tox` i'm not sure what the easiest way is to fix it.

We also don't even have an `all` configuration in `setup.cfg` under `extras_require`  See that we refer to `all` here in tox.ini: https://github.com/spacetelescope/astrocut/blob/main/tox.ini#L61 but there's no all config in extras_require

One possible fix would be to include `docs` for `alldeps`. I've not tested this but it seems like it might work. Change:

`alldeps: all`

to 

`alldeps: docs`

here: https://github.com/spacetelescope/astrocut/blob/main/tox.ini#L61

Reply:
OK will do that now. Quick q: Originally the `alldeps` was set to `all` - if the `all` doesn't exist, what was it doing? I imagine it would break with some sort of message saying `all` is not a thing

Reply:
i have no idea.

i may also have this backwards and it's "setting" to `all` was actually only a description of "alldeps". which would still not make sense, because i don't know how it was pulling in the docs dependencies there, either.

I think this will require some investigation into tox configuration and `extras_require`

maybe @pllim can give us a few ideas?

Reply:
Seems like this suggestion got tox to install `sphinx-astropy`.

still a mystery to me how this was ever working before, when we had no `all` config.

Reply:
You only use alldeps if you have `all` defined. Otherwise, it means nothing to your CI.

Without knowing the history of your code, I cannot tell you why it was working before. "alldeps" does not automatically grab "all". "alldeps" is just a name, and you can make it install whatever you want, really.

Reply:
Thanks! This was my intuition, as well.